From 063bd2152faa8fda64f76ddc9e0128f10445f00c Mon Sep 17 00:00:00 2001
From: LN Liberda <lauren@selfisekai.rocks>
Date: Mon, 22 Dec 2025 00:59:00 +0100
Subject: [PATCH] musl/linux: Don't import conflicting libc and kernel headers

/usr/include/sys/prctl.h:88:8: error: redefinition of 'prctl_mm_map'
/usr/include/linux/prctl.h:134:8: note: previous definition is here

Change-Id: Id2fa78a480d622f221a7dedf875481f23fc21d29
---

--- a/third_party/libwebrtc/rtc_base/platform_thread_types.cc
--- b/third_party/libwebrtc/rtc_base/platform_thread_types.cc
@@ -12,7 +12,9 @@
 
 // IWYU pragma: begin_keep
 #if defined(WEBRTC_LINUX)
+#if defined(__GLIBC__)
 #include <linux/prctl.h>
+#endif
 #include <sys/prctl.h>
 #include <sys/syscall.h>
 
